### PR TITLE
adds .taprc file support #228

### PR DESCRIPTION
**Pull request by kgraves**

### CODE REVIEW

File: bin/run.js [hunk 1]
```diff
@@ -93,26 +107,37 @@ function main () {
...
   }
 
-  options.color = supportsColor
   if (process.env.TAP_COLORS !== undefined) {
-    options.color = !!(+process.env.TAP_COLORS)
+    defaultArgs.color = !!(+process.env.TAP_COLORS)
+  }
+
+  if (process.env.TAP === '1') {
+    defaultArgs.reporter = 'tap'
```
Comment (isaacs) [inline bin/run.js]: This should actually be set on the final option object, not just the default set.

File: test/utils.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  })
+
+  t.test('returns empty object when not valid yaml', function (t) {
+    var expected = {
+      timeout: 9999,
+      coverage: false,
+      coverageReport: false,
+      reporter: 'classic'
+    }
+    t.strictSame(utils.parseRcFile(validYmlPath), expected)
```
Comment (isaacs) [inline test/utils.js]: Should this be checking a file that's not valid yaml?  Maybe use `__filename` or something?

**Comment by isaacs** [comment]
Landed and published on 6.0.0.  thanks!

**Comment by kgraves** [comment]
woohoo :+1:

**Pull request closed without merging**